variables.tf

```diff
@@ -63,3 +65,18 @@ variable "external_domain_name" {
 variable "route53_external_id" {
   description = "DNS external name id"
 }
+
+variable "health_check_conf" {
+  description = "Parameters to configure health checks for the load balancer target group"
+  type = "map"
+
+  default = {
+    interval = "30"
+    path = "/"
+    port = "traffic-port"
```

Comment:
I'm not too familiar with this attribute, but is this normal? Where is 'traffic-port' defined?

Reply:
That's an argument to amazon to tell it to use the same port as the protocol

Reply:
Neat, thanks for the info!

Reply:
How would this code affect load balancers that don't need it?

Reply:
It doesn't do anything in that case - I've set the default params to match each default that TF/Amazon sets when it makes an LB.